doc/release_notes_0.2.rst

```diff
@@ -17,6 +17,21 @@ ACRN device model, and documentation.
 Version 0.2 new features
 ************************
 
+Fixed Issues
+============
+:acrn-issue:`663` - Black screen displayed after booting SOS/UOS
```

Comment:
The intention was to also add a description of the issue and if there is a workaround (see the 0.1 release notes for an example).

Each of these issues needs to have a blank line between them (or they'll run together as one paragraph).  (Click on the VIEW button here in GitHub to see what I mean.)

Reply:
The VIEW button when you're looking at the file changed:  https://github.com/projectacrn/acrn-hypervisor/blob/7fc0fd8bd1f12328bb492ff0f3e891656462f21a/doc/release_notes_0.2.rst

Reply:
doh.  You're right, I was thinking these are Known issues not fixed ones.